src/components/drawer/drawer.spec.js

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
```

Comment:
Hi I can see some things are not tested:
- aria attributes
- data-element
I think It would be good to have these tests

Reply:
updated